addon/sidebar.css

```diff
@@ -0,0 +1,28 @@
+body, html {
+  height: 100%;
+  width: 100%;
+}
+#onboarding {
+  margin: 1em;
+}
+#browser-container {
+  width: 100%;
+  height: 100%;
+}
+iframe {
+  width: 100%;
+  height: 100%;
+  border: none;
+}
+#iframe-container {
+  height: 100%;
+  width: 100%;
+  padding-bottom: 20px;
+}
+#toolbar {
+  width: 100%;
+  position: absolute;
+  bottom: 0px;
```

Comment:
nit: change to `0` without units.

This validates using [**stylelint**](http://npm.im/stylelint) using the [**stylelint-config-recommended**](http://npm.im/stylelint-config-recommended) ruleset, but has a few errors if we want to use the more strict-er [**stylelint-config-standard**](http://npm.im/stylelint-config-standard) rules:

```sh
> $(npm bin)/stylelint 'addon/**/*.css'

addon/sidebar.css
  1:5   ✖  Expected newline after ","        selector-list-comma-newline-after
  5:1   ✖  Expected empty line before rule   rule-empty-line-before
  8:1   ✖  Expected empty line before rule   rule-empty-line-before
 12:1   ✖  Expected empty line before rule   rule-empty-line-before
 17:1   ✖  Expected empty line before rule   rule-empty-line-before
 22:1   ✖  Expected empty line before rule   rule-empty-line-before
 25:12  ✖  Unexpected unit                   length-zero-no-unit
```

I'm fine with merging this as-is and I can add **stylelint** later, if we think it adds value (which it probably doesn't if we don't plan on changing CSS in the future).

Reply:
Yeah, this commit is just a straight copy of the CSS. We should do stylelint separately.